code/game/objects/items/devices/sound_synth.dm

```diff
@@ -0,0 +1,64 @@
+/obj/item/device/soundsynth
+	name = "sound synthesizer"
+	desc = "A device that is able to create sounds."
+	icon_state = "radio"
+	icon = 'icons/obj/device.dmi'
+	w_class = WEIGHT_CLASS_TINY
+
+	var/tmp/spam_flag = 0 //To prevent mashing the button to cause annoyance like a huge idiot.
```

Comment:
tmp?

Reply:
just use the cooldown defines for this not weird stuff

Reply:
Which cooldown should I use? I think I found the defines but none of them seem to be 2 seconds long.

If I'm supposed to use the defines to create a 2 second long cooldown (or longer), what exactly is the proper syntax to do that?